roles/openshift_excluder/tasks/status.yml

```diff
@@ -20,13 +20,15 @@
 - name: Update to latest excluder packages
   package:
     name: "{{ openshift.common.service_type }}-excluder"
+    state: latest
```

Comment:
After further analysis:
1) `playbooks/byo/config.yml` includes `playbooks/common/openshift-cluster/config.yml`
2) `playbooks/common/openshift-cluster/config.yml` includes `playbooks/common/openshift-cluster/disable_excluder.yml`
3) `playbooks/common/openshift-cluster/disable_excluder.yml` includes `roles/openshift_excluder/tasks/status.yml`

So the excluders are used during installation as well.